cli/installer/install-azd.ps1

```diff
@@ -251,42 +255,51 @@ function reportTelemetryIfEnabled($eventName, $reason='', $additionalProperties
     }
 }
 
-try {
-    if (isLinuxOrMac -and !$InstallFolder) {
-        $InstallFolder = "/usr/local/bin"
+if (isLinuxOrMac) {
+    if (!(Get-Command curl)) { 
+        Write-Error "Command could not be found: curl."
+    }
+    if (!(Get-Command bash)) { 
+        Write-Error "Command could not be found: bash."
     }
 
-    $binFilename = ''
-    $extension = 'msi'
-    $packageFilename = "azd-windows-amd64.$extension"
+    $params = @(
+        '--base-url', $BaseUrl, 
+        '--version', "'$Version'"
+    )
 
-    if (isLinuxOrMac) {
-        $platform = 'linux'
-        $extension = 'tar.gz'
-        if ($IsMacOS) {
-            $platform = 'darwin'
-            $extension = 'zip'
-        }
+    if ($InstallFolder) {
+        $params += '--install-folder', $InstallFolder
```

Comment:
Does this work with install / symlink folders having spaces?

Reply:
When it gets splatted to a native command, no. You should quote it:
```suggestion
        $params += '--install-folder', "$InstallFolder"
```

Reply:
Same thing anywhere else below, just to be safe.

Reply:
After testing it does need to be: 

```
"`"$InstallFolder`""
```

Thanks, though. Quotes do belong around all those parameters!